src/api/records.tsx

```diff
@@ -98,15 +102,25 @@ const fetchRecordCountQuery = (
 
 // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-constraint
 export const useRecords = <T extends unknown = Record[]>(
-  options?: UseQueryOptions<Record[], AxiosError, T, [string]>
+  options?: UseQueryOptions<
+    Record[],
+    AxiosError,
+    T,
+    [
+      string,
+      {
+        filters: string[];
+      }
+    ]
+  >
 ): UseQueryResult<T, AxiosError> => {
-  useAppSelector(selectQueryParams);
+  const { filters } = useAppSelector(selectQueryParams);
```

Comment:
Just double checking - do we want the filters used in this function? I'm in the middle of changing this in my PR anyway because this function is only being used in plotting at the moment but what this means is that filters on the table also impact plot data. This also means that if a user draws a plot, minimizes the window, applies some filters on the table and opens the plot back up again, they may well find it's changed since they last looked at it!

Reply:
This is how it works in eCat and how I think they want things to work. i.e. they use the table to select data and that is what is used in the plots

Reply:
That's fine, just making sure